internal/envbuilder/builder.go

```diff
@@ -52,55 +57,105 @@ type PromptOption struct {
 
 type ParsedYaml map[string][]UserPrompt
 
-func GatherUserPrompts(rootDir string) ([]UserPrompt, []string, error) {
+// It will render as:
+//
+//	# The path to the VertexAI application credentials JSON file.
+//	VERTEXAI_APPLICATION_CREDENTIALS=whatever-user-entered
+func (up UserPrompt) String() string {
+	result := ""
+
+	// Disabling it for now as it duplicates comments on every .env file save
+	//if up.Help != "" {
+	//	result += fmt.Sprintf("\n# %v\n", up.Help)
+	//}
```

Comment:
```suggestion
	// TODO Disabling it for now as it duplicates comments on every .env file save
	//if up.Help != "" {
	//	result += fmt.Sprintf("\n# %v\n", up.Help)
	//}
```

As a reminder to find this again later.

Reply:
I already have related change [here](https://github.com/datarobot-oss/cli/pull/152/files#diff-50a21cbbcdebb2f32d4205a2d752321cc75e774ebfb3e72270d50dc20c84154bR67-R69). I think that was main reason for this refactor.